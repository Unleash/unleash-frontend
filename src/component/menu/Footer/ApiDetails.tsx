import { ReactElement } from 'react';
import ConditionallyRender from '../../common/ConditionallyRender';
import { IVersionInfo } from '../../../interfaces/uiConfig';
import { formatCurrentVersion, formatUpdateNotification } from './ApiDetails.helpers';

export interface IPartialUiConfig {
	name: string;
	version: string;
	slogan?: string;
	environment?: string;
	versionInfo?: IVersionInfo;
}

interface IApiDetailsProps {
	uiConfig: IPartialUiConfig;
}

export const ApiDetails = (props: IApiDetailsProps): ReactElement => {
	const instanceId = props.uiConfig.versionInfo?.instanceId;
	const currentVersion = formatCurrentVersion(props.uiConfig);
	const environment = props.uiConfig.environment;
	const updateNotification = formatUpdateNotification(props.uiConfig);
	
	return (
		<section title="API details">
			<h4>
				{currentVersion}{' '}
				<ConditionallyRender
					condition={Boolean(environment)}
					show={<small>({environment})</small>}
				/>
			</h4>
			<ConditionallyRender
				condition={Boolean(updateNotification)}
				show={
					<small>
						{updateNotification}
						<br />
					</small>
				}
			/>
			<br />
			<small>{props.uiConfig.slogan}</small>
			<br />
			<ConditionallyRender
				condition={Boolean(instanceId)}
				show={<small>{`${instanceId}`}</small>}
			/>
		</section>
	);
};
