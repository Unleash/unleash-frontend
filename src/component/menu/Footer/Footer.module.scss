.list {
    padding: 0;
    margin: 0;
}

.listItem {
    padding: 0;
    margin: 0;
}

.listItem a {
    text-decoration: none;
    color: #fff;
}