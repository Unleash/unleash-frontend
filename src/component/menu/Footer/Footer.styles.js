import { makeStyles } from '@material-ui/styles';

export const useStyles = makeStyles(theme => ({
    footer: {
        background: theme.palette.footer.background,
        padding: '2rem 4rem',
        width: '100%',
        flexGrow: 1,
        zIndex: 100,
    },
    list: {
        padding: 0,
        margin: 0,
    },
    listItem: {
        padding: 0,
        margin: 0,
        '& a': {
            textDecoration: 'none',
            color: theme.palette.footer.main,
        },
    },
}));
