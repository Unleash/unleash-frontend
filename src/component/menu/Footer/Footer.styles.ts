import makeStyles from '@mui/styles/makeStyles';

export const useStyles = makeStyles(theme => ({
    footer: {
        // @ts-expect-error
        background: theme.palette.footer.background,
        padding: '2rem 4rem',
        width: '100%',
        flexGrow: 1,
        zIndex: 100,
    },
    list: {
        padding: 0,
        margin: 0,
    },
    listItem: {
        padding: 0,
        margin: 0,
        '& a': {
            textDecoration: 'none',
            // @ts-expect-error
            color: theme.palette.footer.main,
        },
    },
}));
