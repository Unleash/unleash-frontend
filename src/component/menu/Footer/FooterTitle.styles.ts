import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
    title: {
        all: 'unset',
        display: 'block',
        margin: '1rem 0',
        fontSize: '1rem',
        fontWeight: theme.fontWeight.bold,
    },
}));
