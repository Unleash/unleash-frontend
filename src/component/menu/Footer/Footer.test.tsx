import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { ThemeProvider, Theme, StyledEngineProvider } from '@mui/material';

import Footer from './Footer';
import theme from 'themes/mainTheme';


declare module '@mui/styles/defaultTheme' {
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  interface DefaultTheme extends Theme {}
}


test('should render DrawerMenu', () => {
    const tree = renderer.create(
        <StyledEngineProvider injectFirst>
            <ThemeProvider theme={theme}>
                <MemoryRouter>
                    <Footer />
                </MemoryRouter>
            </ThemeProvider>
        </StyledEngineProvider>
    );

    expect(tree).toMatchSnapshot();
});

test('should render DrawerMenu with "features" selected', () => {
    const tree = renderer.create(
        <StyledEngineProvider injectFirst>
            <ThemeProvider theme={theme}>
                <MemoryRouter initialEntries={['/features']}>
                    <Footer />
                </MemoryRouter>
            </ThemeProvider>
        </StyledEngineProvider>
    );

    expect(tree).toMatchSnapshot();
});
