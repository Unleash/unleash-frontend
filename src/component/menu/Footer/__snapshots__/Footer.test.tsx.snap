// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DrawerMenu 1`] = `
<footer
  className="makeStyles-footer-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 css-16chest-MuiGrid-root"
    style={
      Object {
        "marginBottom": 0,
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root"
    >
      <section
        title="API details"
      >
        <h2
          className="makeStyles-title-4"
        >
          Unleash 3.x
           
        </h2>
        <br />
        <small>
          The enterprise ready feature toggle service.
        </small>
        <br />
      </section>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-auto css-lvo4zz-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-7 css-3f6ufr-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <section
            title="Unleash SDK"
          >
            <h2
              className="makeStyles-title-4"
            >
              Server SDKs
            </h2>
            <ul
              className="MuiList-root MuiList-padding MuiList-dense makeStyles-list-2 css-h4y409-MuiList-root"
            >
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/node_sdk"
                      target="_blank"
                    >
                      Node.js
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/java_sdk"
                      target="_blank"
                    >
                      Java
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/go_sdk"
                      target="_blank"
                    >
                      Go
                    </a>
                  </span>
                </div>
              </li>
               
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/ruby_sdk"
                      target="_blank"
                    >
                      Ruby
                    </a>
                  </span>
                </div>
              </li>
               
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/python_sdk"
                      target="_blank"
                    >
                      Python
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/dot_net_sdk"
                      target="_blank"
                    >
                      .NET
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/php_sdk"
                      target="_blank"
                    >
                      PHP
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks"
                      target="_blank"
                    >
                      All SDKs
                    </a>
                  </span>
                </div>
              </li>
            </ul>
          </section>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <section
            title="Unleash SDK"
          >
            <h2
              className="makeStyles-title-4"
            >
              Frontend SDKs
            </h2>
            <ul
              className="MuiList-root MuiList-padding MuiList-dense makeStyles-list-2 css-h4y409-MuiList-root"
            >
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/unleash-proxy"
                      target="_blank"
                    >
                      Unleash Proxy
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/proxy-javascript"
                      target="_blank"
                    >
                      JavaScript SDK
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/proxy-react"
                      target="_blank"
                    >
                      React SDK
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/proxy-ios"
                      target="_blank"
                    >
                      iOS SDK
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/android_proxy_sdk"
                      target="_blank"
                    >
                      Android SDK
                    </a>
                  </span>
                </div>
              </li>
            </ul>
          </section>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <section>
            <h2
              className="makeStyles-title-4"
            >
              About
            </h2>
            <ul
              className="MuiList-root MuiList-padding MuiList-dense makeStyles-list-2 css-h4y409-MuiList-root"
            >
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://www.getunleash.io/"
                      target="_blank"
                    >
                      getunleash.io
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://twitter.com/getunleash"
                      target="_blank"
                    >
                      Twitter
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://www.linkedin.com/company/getunleash"
                      target="_blank"
                    >
                      LinkedIn
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://github.com/Unleash/unleash"
                      target="_blank"
                    >
                      GitHub
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://slack.unleash.run"
                      target="_blank"
                    >
                      Slack Community
                    </a>
                  </span>
                </div>
              </li>
            </ul>
          </section>
        </div>
      </div>
    </div>
  </div>
</footer>
`;

exports[`should render DrawerMenu with "features" selected 1`] = `
<footer
  className="makeStyles-footer-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 css-16chest-MuiGrid-root"
    style={
      Object {
        "marginBottom": 0,
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root"
    >
      <section
        title="API details"
      >
        <h2
          className="makeStyles-title-4"
        >
          Unleash 3.x
           
        </h2>
        <br />
        <small>
          The enterprise ready feature toggle service.
        </small>
        <br />
      </section>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-auto css-lvo4zz-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-7 css-3f6ufr-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <section
            title="Unleash SDK"
          >
            <h2
              className="makeStyles-title-4"
            >
              Server SDKs
            </h2>
            <ul
              className="MuiList-root MuiList-padding MuiList-dense makeStyles-list-2 css-h4y409-MuiList-root"
            >
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/node_sdk"
                      target="_blank"
                    >
                      Node.js
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/java_sdk"
                      target="_blank"
                    >
                      Java
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/go_sdk"
                      target="_blank"
                    >
                      Go
                    </a>
                  </span>
                </div>
              </li>
               
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/ruby_sdk"
                      target="_blank"
                    >
                      Ruby
                    </a>
                  </span>
                </div>
              </li>
               
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/python_sdk"
                      target="_blank"
                    >
                      Python
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/dot_net_sdk"
                      target="_blank"
                    >
                      .NET
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/php_sdk"
                      target="_blank"
                    >
                      PHP
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks"
                      target="_blank"
                    >
                      All SDKs
                    </a>
                  </span>
                </div>
              </li>
            </ul>
          </section>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <section
            title="Unleash SDK"
          >
            <h2
              className="makeStyles-title-4"
            >
              Frontend SDKs
            </h2>
            <ul
              className="MuiList-root MuiList-padding MuiList-dense makeStyles-list-2 css-h4y409-MuiList-root"
            >
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/unleash-proxy"
                      target="_blank"
                    >
                      Unleash Proxy
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/proxy-javascript"
                      target="_blank"
                    >
                      JavaScript SDK
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/proxy-react"
                      target="_blank"
                    >
                      React SDK
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/proxy-ios"
                      target="_blank"
                    >
                      iOS SDK
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://docs.getunleash.io/sdks/android_proxy_sdk"
                      target="_blank"
                    >
                      Android SDK
                    </a>
                  </span>
                </div>
              </li>
            </ul>
          </section>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <section>
            <h2
              className="makeStyles-title-4"
            >
              About
            </h2>
            <ul
              className="MuiList-root MuiList-padding MuiList-dense makeStyles-list-2 css-h4y409-MuiList-root"
            >
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://www.getunleash.io/"
                      target="_blank"
                    >
                      getunleash.io
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://twitter.com/getunleash"
                      target="_blank"
                    >
                      Twitter
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://www.linkedin.com/company/getunleash"
                      target="_blank"
                    >
                      LinkedIn
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://github.com/Unleash/unleash"
                      target="_blank"
                    >
                      GitHub
                    </a>
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding makeStyles-listItem-3 css-1vgy6yu-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-tp0hw6-MuiTypography-root"
                  >
                    <a
                      href="https://slack.unleash.run"
                      target="_blank"
                    >
                      Slack Community
                    </a>
                  </span>
                </div>
              </li>
            </ul>
          </section>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
